model_compression_toolkit/gptq/keras/quantization_facade.py

```diff
@@ -92,14 +92,15 @@ def get_keras_gptq_config(n_iter: int,
             The configuration can be passed to :func:`~model_compression_toolkit.keras_post_training_quantization` in order to quantize a keras model using gptq.
 
         """
-        bias_optimizer = tf.keras.optimizers.SGD(learning_rate=LR_BIAS_DEFAULT)
-        optimizer_quantization_parameter = tf.keras.optimizers.Adam(learning_rate=LR_QUANTIZATION_PARAM_DEFAULT)
+        bias_optimizer = tf.keras.optimizers.SGD(learning_rate=LR_BIAS_DEFAULT, momentum=0.9)
+        optimizer_quantization_parameter = tf.keras.optimizers.SGD(learning_rate=LR_QUANTIZATION_PARAM_DEFAULT, momentum=0.9)
```

Comment:
Please use a constant for momentum value.
Also, it may be worth considering taking all GPTQ constants to a dedicated file (in a different PR).